enatega-multivendor-app/src/components/Main/TopBrands/TopBrands.js

```diff
@@ -85,7 +93,7 @@ function TopBrands(props) {
             activeOpacity={0.8}
             onPress={() => {
               navigation.navigate('Menu', {
-                selectedType: 'restaurant',
```

Comment:
what's the purpose of removing restaurant from here? Below you are sending grocery as selected type but here its empty

Reply:
because here we have to display all brands the restuarants and groceries as well.